automations/js/src/count_user_reviewable_prs.js

```diff
@@ -49,39 +49,46 @@ query ($repoOwner: String!, $repo: String!, $cursor: String) {
     '🟥 priority: critical',
   ]
   const [owner, repo] = GITHUB_REPOSITORY.split('/')
+  const isRelevantPRFromGraphQL = (pr) =>
+    pr.author.login === context.actor &&
+    !pr.isDraft &&
+    !pr.labels.nodes.some((label) => ignoredLabels.includes(label.name))
+  const isRelevantPRFromContext = (pr) =>
+    !pr.draft && !pr.labels.some((label) => ignoredLabels.includes(label.name))
 
   try {
     let hasNextPage = true
     let cursor = null
     let reviewablePRs = []
+    const pullRequest = context.payload.pull_request
```

Comment:
This should be fine based on the shape of `Context`: https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts